src/node.cc

```diff
@@ -2834,6 +2837,13 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "traceDeprecation", True(env->isolate()));
   }
 
+  // --warn-on-sync
+  if (warn_on_sync) {
+    READONLY_PROPERTY(process, "warnOnSync", True(env->isolate()));
```

Comment:
It seems weird that this property is always either true or doesn't exist, should it just always be defined as either true or false?